monai/data/meta_tensor.py

```diff
@@ -280,6 +280,27 @@ def __torch_function__(cls, func, types, args=(), kwargs=None) -> Any:
         ret = MetaTensor.update_meta(ret, func, args, kwargs)
         return ret[0] if unpack else ret
 
+    def __array_function__(self, func, types, args, kwargs):
+        """for numpy Interoperability"""
+        if not func.__module__.startswith("numpy"):
+            return NotImplemented
+        if not any(issubclass(t, MetaTensor) for t in types):
+            # Defer to any other subclasses that implement __array_function__
+            return NotImplemented
+        _args = list(
```

Comment:
I would suggest putting conversion into an inner function, also should `torch.Tensor` objects be converted as well?

```python
def _convert(x):
    if isinstance(x, (MetaTensor, tuple, list)):  # should include torch.Tensor?
        return convert_data_type(x, output_type=np.ndarray, wrap_sequence=False)[0]
    else:
        return x

_args=list(map(_convert, args)
_kwargs={k: _convert(v) for k, v in kwargs.items()}
```